src/lib/select/select.spec.ts

```diff
@@ -988,37 +988,50 @@ describe('MdSelect', () => {
         select.style.marginRight = '30px';
       });
 
-      it('should align the trigger and the selected option on the x-axis in ltr', () => {
-        trigger.click();
-        fixture.detectChanges();
+      it('should align the trigger and the selected option on the x-axis in ltr', async(() => {
+        fixture.whenStable().then(() => {
+          fixture.detectChanges();
```

Comment:
Doesn't look like it, although most of these `whenStable` calls shouldn't be necessary either.